cmd/clean.go

```diff
@@ -79,3 +102,66 @@ func (r *CleanCommand) Run(inputPath string) error {
 
 	return nil
 }
+
+func (r *CleanCommand) cleanUvCache() error {
+	uvChecker := &python.UvChecker{}
+	ctx := context.Background()
+	uvBinaryPath, err := uvChecker.CheckUvInstalled(ctx)
+
+	if err != nil {
+		// Prompt user to install uv if not found
+		if r.confirmUvInstall() {
+			// Use EnsureUvInstalled which will handle the installation
+			uvBinaryPath, err = uvChecker.EnsureUvInstalled(ctx)
+			if err != nil {
+				return errors.Wrap(err, "failed to install uv")
+			}
+		} else {
+			return errors.New("uv is not installed. Please install uv first using the bruin installation process")
+		}
```

Comment:
if uv doesn't exist simply inform the user and move on, there's no need to install uv just to be able to clean its cache.